sdk/python/kfp/cli/component.py

```diff
@@ -362,9 +360,9 @@ def component():
     is_flag=True,
     default=True,
     help='Push the built image to its remote repository.')
-def build(components_directory: pathlib.Path, component_filepattern: str,
-          engine: str, kfp_package_path: Optional[pathlib.Path],
-          overwrite_dockerfile: bool, push_image: bool):
+def build(components_directory: str, component_filepattern: str, engine: str,
+          kfp_package_path: Optional[str], overwrite_dockerfile: bool,
+          push_image: bool):
```

Comment:
nit: (just a small tip, you don't necessarily need to make a change) if you add a comma after the last parameter, formatting would be more predictable--each parameter on their own line.

Reply:
Thanks, @chensun. That's a great tip -- I will keep that in mind for future PRs.